pkg/network/ebpf/c/prebuilt/http2.h

```diff
@@ -0,0 +1,88 @@
+#include "kconfig.h"
```

Comment:
1. why is the file in `prebuilt`? is should be a common file
2. you are missing `ifndef` at the beginning of the file